src/server.ts

```diff
@@ -41,8 +41,7 @@ let linter: any;
 
 // "config"
 let configResolver: ConfigResolver;
-let needUpdateConfig = true;
-let browsersListCache: string[] = [];
+let browsersListCache: { [key: string]: string[] } = {};
```

Comment:
We can use the built-in generic for this type:

```ts
let browsersListCache: Record<string, string[]>;
```

Also, now I prefer to use [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) for hashes, but it if you wanted.